Assets/HoloToolkit/Utilities/Tests/StabilizationPlaneSetting.unity

```diff
@@ -0,0 +1,1372 @@
+%YAML 1.1
```

Comment:
Looking at the scene it seems it's purpose is to test the StablizationPlaneModifier, which is under Utilities. In which case I think the Utilities\Tests folder makes sense.